internal/app/wwctl/container/exec/main.go

```diff
@@ -101,6 +101,10 @@ func CobraRunE(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	// getting syncuser from cmd, e.g., shell comamnd will call exec command and passing --syncuser
+	if syncUserFlag, err := cmd.Flags().GetBool("syncuser"); err == nil {
+		SyncUser = SyncUser || syncUserFlag
+	}
```

Comment:
I don't think this should be necessary; `SyncUser` should already be the value from `--syncuser`.